### PR TITLE
Added cables

### PR DESCRIPTION
**Pull request by macaquedev**

### CODE REVIEW

File: solutions/bio2/Cables/index.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        However, you must be careful to output the points in the correct order,
+        as a loop must be formed. To do this, you can pick the bottom-left-most point 
+        in each group and output the points in anticlockwise order. To do this,
+        you can sort the points in each group by their angle with respect to the
+        bottom-left-most point, which is equal to $\arctan(\frac{brace("y - y_0")}{brace("x - x_0")})$.
+        Make sure to use the C++ function <SCode>atan2(y, x)</SCode> to handle
+        the case where $x$ and/or $y$ are negative.
+      </SText>
+      <STitle>Code</STitle>
+      <SCodeBlock path="cables/sol" />
```
Comment (myst-6) [inline solutions/bio2/Cables/index.tsx:51]: The solution isn't showing up on the preview, did you forget to add it?

**Review by myst-6 — CHANGES_REQUESTED** [review]
missing code

**Comment by macaquedev** [comment]
@myst-6 please accept now

**Review by myst-6 — APPROVED** [review]
had to fix merge conflicts so, please blindly accept or double-check this @AG-88301 🙂

**Review by AG-88301 — APPROVED** [review]
works fine

**Pull request merged**